### ISSUE TITLE
"Debug" Feature not Found

### ISSUE THREAD

**Issue by jakesboy2** [maciejhirsz/logos#408]
Enabling "debug" in the features as shown below does not work.

Cargo.toml
```
logos = { version = "0.14.0", features = ["debug"] }
```

Results in the following output
```
$ cargo build
    Updating crates.io index
error: failed to select a version for `logos`.
    ... required by package `example_project v0.1.0 (/redacted/file/path)`
versions that meet the requirements `^0.14.0` are: 0.14.0

the package `example_project` depends on `logos`, with features: `debug` but `logos` does not have these features.


failed to select a version for `logos` which could resolve this conflict
```

Shown below is `cargo add` output, which also does not list debug as a feature.
```
$ cargo add --dry-run logos
    Updating crates.io index
      Adding logos v0.14.0 to dependencies.
             Features:
             + export_derive
             + logos-derive
             + std
```

Debug is listed as a feature in `cargo.toml` on the main branch, so I'm not sure why it isn't found when I attempt to use it.

**Comment by eikopf** [comment]
I ran into this as well; as far as I can tell, the documentation is _ahead_ of the latest version of `logos` on crates.io. Trawling through issues didn't help much, but it's clear that the debugging section in the book was added about a month ago, while `v0.14.0` was published about 6 months ago.

[This](https://github.com/maciejhirsz/logos/pull/379#issuecomment-2144054139) solution worked for me, though it's obviously not ideal. Kinda just seems like we have to wait for a new release 🙃.

**Comment by jakesboy2** [comment]
Seems like that does the trick, thank you. This will do until theres a release.

**Comment by jeertmans** [comment]
Sorry for the delay between releases, I have just released [v0.14.1](https://github.com/maciejhirsz/logos/releases/tag/v0.14.1) with this feature :-)

**Comment by jakesboy2** [comment]
Incredible, thank you ❤️
